web/src/views/SubmissionPortal/Components/SubmissionDocsLink.vue

```diff
@@ -0,0 +1,24 @@
+<script lang="ts">
+import { defineComponent } from '@vue/composition-api';
+
+export default defineComponent({
+  props: {
+    documentationUrl: {
+      type: String,
+      default: 'https://nmdc-documentation.readthedocs.io/en/latest/howto_guides/submit2nmdc.html',
+    },
+  },
+});
+</script>
+
+<template>
+  <a
+    class="ml-2"
+    :href="documentationUrl"
+    target="_blank"
+  >
+    <v-icon>
+      mdi-information-outline
+    </v-icon>
+  </a>
```

Comment:
I think this needs a tooltip and either different styling or different placement. Its currently hard to notice.

Reply:
Let me know what you think of the changes in 5e32d5e